src/app/page.tsx

```diff
@@ -7,7 +7,8 @@ import PageContainer from "@/components/PageContainer";
 import SearchBar from "@/components/Searchbar";
 import { useSearchParams } from "next/navigation";
 import bgBackground from "../public/bg-background.png";
```

Comment:
important: `src/public/bg-background.png` is too big, please optime the size.

Reply:
it's been already done